### DISCUSSION TITLE
v42.0

### DISCUSSION THREAD

**Discussion by westnordost** [streetcomplete/StreetComplete#3915] [Release Discussions]
<h3>General Enhancements</h3>
  <ul>
  <li>Quest titles are now shorter and more consistent (#3870, #3903), thanks @FloEdelmann, @peternewman</li>
  </ul>
  <h3>New Quests</h3>
  <ul>
  <li><i>Is smoking allowed anywhere in this place?</i> (#539, #3784, #3856, #3865), by @mnalis</li>
  <li><i>What is the surface of the sidewalk here?</i> (#1593, #3735), by @arrival-spring</li>
  <li><i>What kind of seating does this place have?</i> (#1176, #3758, #3911), thanks @fatal69100</li>
  </ul>
  <h3>Quest enhancements</h3>
  <ul>
  <li>Parking fee: Add answer option for when there is just a max parking duration (#3841, #981, #102)</li>
  <li>Sidewalk: Also ask when current tagging is incomplete or invalid (#3821), by @arrival-spring</li>
  <li>Tactile paving on crossings: Add answer option for when it exists only one one side (#3844, #3813), by @arrival-spring</li>
  <li>Barrier details: Add answer option for when there is a different type of barrier (#3846, #3563, #3845, #3817), by @mnalis</li>
  <li>Oneway streets: Also ask for roads that are below or equal 4 meters wide</li>
  <li>Cycleway: Warn before removing that cyclists may use a oneway road in both directions (#3795)</li>
  <li>Halal: Don't ask for vegan only places (#3883), by @peternewman</li>
  <li>Fire hydrant position: Allow deletion of hydrant if it doesn't exist (#3871)</li>
  <li>Housenumbers: More convenient input for housename & housenumber, plus fixed various small issues (#3904, #3907)</li>
  </ul>
  <h3>Enhancements and Fixes</h3>
  <ul>
  <li>Fix crash issues on Android 5 (#3889, #3914), thanks @FloEdelmann</li>
  <li>Fix shop type suggestions were not available in Norwegian (Bokmål) and Serbian (#3890, #3896, #3897), thanks @matkoniecz</li>
  <li>Added support for region or script-specific shop type suggestions, e.g. English in New Zealand or Serbian in Latin or Cryllic (#3897)</li>
  <li>Fix rare crashes (#3888)</li>
  <li>Other minor enhancements and fixes (#3852, #3893, #3895, #3867...), thanks @matkoniecz, @mnalis, @Cj-Malone</li>
  </ul>

<hr /><em>This discussion was created from the release <a href='https://github.com/streetcomplete/StreetComplete/releases/tag/v42.0'>v42.0</a>.</em>

**Comment by mnalis** [comment]
Haha, happy Aprils fools' everyone; when did this get in? :smile: 

![small_Screenshot_20220401_004001_de westnordost streetcomplete mn debug](https://user-images.githubusercontent.com/156656/161161429-48ca66c4-93e0-4fa6-b46b-18946050b349.jpg)

**Reply by matkoniecz** [reply]
I remember seeing it quite long ago in commits :)

**Reply by peternewman** [reply]
I think I need to file a bug report, it didn't activate at midnight, only when I relaunched the app on April Fools day! :rofl:
